src/template.js

```diff
@@ -710,80 +710,254 @@ module.exports = {
+                    // prettier-ignore
+                    cond: ['and',
+                        ['==', '$.answers.p-mainapplicant-confirmation-method.q-mainapplicant-confirmation-method', 'text'],
+                        ['|role.all', 'mainapplicant', 'adult', 'incapable']
+                    ],
+                    data: {
+                        templateId: '3e625f9f-75c4-4903-818e-220829bfc2af',
+                        phoneNumber:
+                            '||/answers/p-mainapplicant-confirmation-method/q-mainapplicant-enter-your-telephone-number||',
+                        personalisation: {
+                            caseReference: '||/answers/system/case-reference||'
+                        },
+                        reference: null
+                    }
+                },
+                {
+                    description: 'Confirmation sms - mainapplicant.applicant:child',
+                    type: 'sendSms',
+                    // prettier-ignore
+                    cond: ['and',
+                        ['==', '$.answers.p-mainapplicant-confirmation-method.q-mainapplicant-confirmation-method', 'text'],
+                        ['|role.all', 'mainapplicant', 'child']
+                    ],
+                    data: {
+                        templateId: 'd2185426-2177-4049-a5b1-b9c6b12e1a79',
+                        phoneNumber:
+                            '||/answers/p-mainapplicant-confirmation-method/q-mainapplicant-enter-your-telephone-number||',
+                        personalisation: {
+                            caseReference: '||/answers/system/case-reference||'
+                        },
+                        reference: null
+                    }
+                },
+                {
+                    description: 'Confirmation sms - rep.applicant:adult:capable',
+                    type: 'sendSms',
+                    // prettier-ignore
+                    cond: ['and',
+                        ['==', '$.answers.p-rep-confirmation-method.q-rep-confirmation-method', 'text'],
+                        ['|role.all', 'rep', 'adult', 'capable']
+                    ],
                     data: {
-                        templateId: 'l10nt:templateId{?lng,context,ns}',
-                        phoneNumber: 'l10nt:phoneNumber{?lng,context,ns}',
+                        templateId: 'b51e5e19-f469-4f8a-a5a2-00499da6f027',
+                        phoneNumber:
+                            '||/answers/p-rep-confirmation-method/q-rep-enter-your-telephone-number||',
                         personalisation: {
                             caseReference: '||/answers/system/case-reference||'
                         },
-                        reference: null // required for notify api
+                        reference: null
+                    }
+                },
+                {
+                    description: 'Confirmation sms - rep.mainapplicant.applicant:adult:incapable',
```

Comment:
should we add the `authority` label to this description to be explicit, and not infer it?

`Confirmation sms - rep.mainapplicant:authority.applicant:adult:incapable`